Scripts/init.sh

```diff
@@ -109,6 +111,45 @@ export DOS_THEME_700="E64A19"; #Deep Orange 700
 
 umask 0022;
 
+# by default we will calculate the max CPU count automatically (used by e.g. repo commands)
+# if the required tool "nproc" is not found a default is used.
+export FALLBACK_MAX_THREADS_REPO=4;
+export MAX_THREADS_REPO_RATE=8; #to avoid being rate limited we never go above this for syncing
+export FALLBACK_MAX_THREADS_BUILD="nolimit"; #if nothing specified we will use all CPU power available
+
+calcThreads(){
```

Comment:
this is needless complexity

Reply:
I don't see any needless conplexity here but more flexibility. You might not need it which is ok but i do ;) 

That being said I'm fine if you wanna close this pr